packages/adapter-pg/src/conversion.ts

```diff
@@ -1,9 +1,10 @@
 import { type ColumnType, ColumnTypeEnum, JsonNullMarker } from '@prisma/driver-adapter-utils'
-import * as pg from 'pg'
+// @ts-ignore: this is used to avoid the `Module '"<path>/node_modules/@types/pg/index"' has no default export.` error.
+import pg from 'pg'
 import { parse as parseArray } from 'postgres-array'
 
-const types = pg.types
-const ScalarColumnType = types.builtins
+const { types } = pg
+const { builtins: ScalarColumnType, getTypeParser, setTypeParser } = types
```

Comment:
This is the important change. `@prisma/adapter-pg-worker/src/conversion.ts` is basically a copy-paste of the edits to this file.